doc/modules/timer.md

```diff
@@ -0,0 +1,33 @@
+# Timer
+
+
+## Summary
+
+Weex encapulates a series of APIs in order to start/stop a one time task or a repeated task at a fixed delay. Please be noted that this moudle **don't provide an accuracy dealy**. It provides best-effort delivery, but the actual dealy may still exceed the delay user wants if the corresponding thread is busy.
```

Comment:
* " a one time " -> "a timer?"
* " Please be noted that this moudle **don't provide an accuracy dealy**" -> " Please note that this module **don't provide an accuracy delay**"